site/src/components/form.md

```diff
@@ -43,58 +43,226 @@ It can be used to hold `.field-body` containers, which holds the individual elem
+		</label>
+		<span class="field-description" id="sample-input-description">
 			Optional field description
 			<!-- 👈 optional -->
-		</p>
+		</span>
 		<div class="field-body">
 			<input
-				id="sample-form-validation-input"
-				name="sample-input"
+				id="sample-input-min"
+				name="input-min"
 				class="input"
 				type="text"
+				value=""
 				required
+				minlength="10"
+				maxlength="255"
+				aria-describedby="sample-input-description"
+			/>
+		</div>
+	</div>
+	<div class="field">
+		<label class="label" for="sample-select">
+			Select
+			<span class="required-indicator"></span>
+		</label>
+		<span id="sample-select-description" class="field-description"> Select 1 of 3 options. </span>
+		<div class="field-body">
+			<select
+				id="sample-select"
+				name="select"
+				class="select"
+				aria-describedby="sample-select-description"
+			>
+				<option value="1">1</option>
+				<option value="2">2</option>
+				<option value="3">3</option>
+			</select>
+		</div>
+	</div>
+	<div class="field">
+		<label class="label" for="sample-tags">
+			Tags
+			<span class="required-indicator"></span>
+		</label>
+		<span id="sample-tags-description" class="field-description"> Select up to 5 tags. </span>
+		<div class="field-body">
+			<input
+				id="sample-tags"
+				name="tags"
+				class="tag-input input"
+				aria-describedby="sample-select-description"
+				hidden
+				aria-hidden="true"
+				value=""
+				mintags="1"
+				maxtags="5"
 			/>
+			<div class="tag-input-holder margin-bottom-xxs has-height-auto">
+				<div class="tags"></div>
+				<input
+					role="combobox"
+					maxlength="100"
+					id="tag-input"
+					class="autocomplete-input input"
+					type="text"
+					aria-label="Tag selector"
+					placeholder="Enter tags..."
+					hidden
+				/>
+			</div>
+			<button class="add-tags-button" type="button">Add tags</button>
+		</div>
+	</div>
+	<div class="field">
+		<label class="label" for="sample-text-area">
+			Textarea
+			<span class="required-indicator"></span>
+		</label>
+		<div class="field-body">
+			<textarea
+				id="sample-text-area"
+				name="text-area"
+				class="textarea"
+				minlength="10"
+				rows="4"
+				cols="30"
+			>
+			</textarea>
+		</div>
+	</div>
+
+	<div class="field">
+		<div class="field-body">
+			<star-rating id="star-rating-1" name="star-rating-1" required>
+				<legend slot="legend">Star rating</legend>
+				<span slot="label-1">Terrible</span>
+				<span slot="label-2">Poor</span>
+				<span slot="label-3">Fair</span>
+				<span slot="label-4">Good</span>
+				<span slot="label-5">Great</span>
+			</star-rating>
 		</div>
-		<p class="field-error">Input is required.</p>
 	</div>
 	<div class="display-flex">
-		<button class="button button-primary margin-right-xxs">Submit</button>
-		<button class="button button-danger">Cancel</button>
+		<button type="submit" class="button button-primary button-filled">Submit form</button>
+		<button type="button" class="button">Cancel</button>
```

Comment:
might as well remove cancel since it doesn't apply to the example

Reply:
removed